jvm/src/test/java/com/microsoft/bond/TestHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.bond;
+
+import java.lang.reflect.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+/**
+ * Contains helper methods for implementing unit tests.
+ */
+public final class TestHelper {
+
+    // prevent instantiation
+    private TestHelper() {
+    }
+
+    /**
+     * Verifies that the argument class meets a set of informal conditions for a "static class", which are:
+     * <ul>
+     * <li>
+     * The class is a concrete class (not an abstract class or an interface), and
+     * </li>
+     * <li>
+     * The class declares a parameterless constructor with private visibility, and
+     * </li>
+     * <li>
+     * The class doesn't declare any instance members other than the single constructor above, and
+     * </li>
+     * <li>
+     * The class is declared as final.
+     * </li>
+     * </ul>
+     * The method raises an {@link AssertionError} if any of the conditions above are not met. It also invokes
+     * the private parameterless constructor to make sure it's covered by unit tests.
+     *
+     * @param clazz the class to verify
+     */
+    public static void verifyStaticClass(Class<?> clazz) {
```

Comment:
verifyStaticHelperClass

---
In reply to: [119245615](https://github.com/Microsoft/bond/pull/465#discussion_r119245615) [](ancestors = 119245615)